src/app/admin/generator/page.tsx

```
@@ -0,0 +1,409 @@
+                                <div className="flex flex-wrap gap-2">
+                                    {styles.map(style => (
+                                        <div
+                                            key={style.id}
+                                            onClick={() => setSelectedStyle(style.id)}
+                                            className={`
+                                                cursor-pointer px-3 py-1.5 rounded-full text-sm font-medium border transition-all
+                                                ${selectedStyle === style.id
+                                                    ? "bg-primary text-primary-foreground border-primary"
+                                                    : "bg-muted/50 hover:bg-muted border-transparent hover:border-muted-foreground/20"}
+                                            `}
+                                        >
```

Biome (2.3.13)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)